app/workers/publish_data_worker.rb

```diff
@@ -0,0 +1,16 @@
+class PublishDataWorker
```

Comment:
good call, it was. I've updated to just call the workers from the lifecycle now.